drivers/timer/ch32v00x_systick.c

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
usual notes + 1 remark. All drivers names should be prefixed with the manufacturer + any specific when necessary

For instance, this should be

```
wch_xxx or
wch_systick_xxx <-- preferable way for this driver I believe.
systick_wch_xxx <-- some drivers uses the specific at first. Best is to check with driver maintainer.
```
instead a SoC series

`ch32v00x_xxx`

Unfortunately this is not harmonized on Zephyr drivers.

Reply:
still relevant, and I think `wch_systick.c` makes sense? I'd imagine arm parts would use either the cortex systick or a more specific `wch_timer.c` something down the road

Reply:
Yep - the rename is done now. Thanks!